dispatcher/templates/custom.jinja2

```diff
@@ -0,0 +1,2 @@
+<!-- Just takes one parameter and renders that one unescaped -->
+{{ body }}
```

Comment:
Just to recap. This works fine for code claiming in discounts, but not if I try to send the following in the statutory massmailer;
```md
{first_name} - user's first name
{last_name} - user's last name
{participant_type_order} - applicant's type order, in this format: type (order), or not set if it's not set by the board.
{body_name} - name of the body
{statutory_id} - applicant's statutory id


*italic*
**bold**


# Header 1


## Header 2
```

Reply:
This was the resulting error in dispatcher;
```
Traceback (most recent call last):
  File "/usr/app/src/main.py", line 275, in <module>
    main()
  File "/usr/app/src/main.py", line 271, in main
    channel.start_consuming()
  File "/usr/local/lib/python3.9/site-packages/pika/adapters/blocking_connection.py", line 1883, in start_consuming
    self._process_data_events(time_limit=None)
  File "/usr/local/lib/python3.9/site-packages/pika/adapters/blocking_connection.py", line 2044, in _process_data_events
    self.connection.process_data_events(time_limit=time_limit)
  File "/usr/local/lib/python3.9/site-packages/pika/adapters/blocking_connection.py", line 851, in process_data_events
    self._dispatch_channel_events()
  File "/usr/local/lib/python3.9/site-packages/pika/adapters/blocking_connection.py", line 567, in _dispatch_channel_events
    impl_channel._get_cookie()._dispatch_events()
  File "/usr/local/lib/python3.9/site-packages/pika/adapters/blocking_connection.py", line 1510, in _dispatch_events
    consumer_info.on_message_callback(self, evt.method,
  File "/usr/app/src/main.py", line 114, in send_email
    rendered = template.render(msg['parameters'], altro=msg['subject'])
  File "/usr/local/lib/python3.9/site-packages/jinja2/environment.py", line 1286, in render
    ctx = self.new_context(dict(*args, **kwargs))
ValueError: dictionary update sequence element #0 has length 1; 2 is required
```

Reply:
thanks, I will have a look